_topics/en/silent-payments.md

```diff
@@ -73,5 +73,30 @@ excerpt: >
 
 ---
 
+Traditionally, a new address should be generated for every
+payment. This is because when you receive several payments
+to the same Bitcoin address, it is a given that the same
+person or entity received all of those payments even if the outputs are later
+spent in separate transactions. This is also known as address reuse,
+and more can be found under [output linking][topic output linking].
+
+Using a new address often requires a secure interaction between sender
+and receiver so that the receiver can provide a fresh address everytime.
+However, interaction is often infeasible and in many cases undesirable.
+
+With silent payments, a receiver can generate a silent payment address
+and make it publicly known, thus eliminating the need for interaction.
+The sender can select one or more of their chosen inputs and use their
+secret key(s) to derive the shared secret (together with public key of
+the silent payment address), which is used to generate the destination.
+
+The intended recipient detects the payment by scanning eligible transactions
```

Comment:
```suggestion
The intended recipient detects the payment by scanning transactions
```
While I understand why only some transactions are eligible, we're not getting into enough detail here to even hint at what that means, so I think the word "eligible" will leave readers wondering. Also, to be precise, a transaction must be "scanned" to determine if it's eligible in the first place. I know that scanning an ineligible transaction is much less work (it can be eliminated quickly) but perhaps that's a nuance not worth trying to convey here.

Reply:
Thank you for explaining the rationale behind this, updated!